tools/bin/acceptance_test_kube_schedulerv2.sh

```diff
@@ -34,7 +34,7 @@ echo "Listing nodes scheduled for pods..."
 kubectl describe pods | grep "Name\|Node"
 
 # allocates a lot of time to start kube. takes a while for postgres+temporal to work things out
-sleep 120s
+sleep 120
```

Comment:
curious, was this breaking anything? or just formatting?

Reply:
breaks on some environments that don't parse the s locally